noodles-editor/src/noodles/components/node-properties.tsx

```diff
@@ -202,7 +202,7 @@ function NodeProperties({ node }: { node: NodeJSON<unknown> }) {
       <div className={s.header}>
         <div className={s.title}>
           {displayName}
-          <div className={cx(s.capsule, headerClass(name))}>{typeCategory(name)}</div>
+          <div className={cx(s.capsule, headerClass(type))}>{typeCategory(type)}</div>
```

Comment:
So why does this work in production currently?

Reply:
The only parts that work in production are using `const opTypes = {}` map which is stringified before minification. Nothing in production referencing `constructor.name` are working - they're just subtle issues

Reply:
<img width="287" height="47" alt="Image" src="https://github.com/user-attachments/assets/de0b47d9-db5d-4807-bb87-e47436afd228" />

The badges are all "Unknown"

Reply:
Ah. And what are the node headers using / why does that work?

Reply:
Looked into it and apparently they're using `name` from the constructor